lib/screens/app/profile/profile_screen.dart

```diff
@@ -0,0 +1,511 @@
+          ListTile(
+            leading: Icon(
+              Icons.favorite_border,
+              color: AppColors.springGreen,
+            ),
+            title: Text(
+              'Skills & Interest'.i18n,
+              style: Theme.of(context).textTheme.subtitle2HighEmphasis,
+            ),
+          ),
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Wrap(
+              spacing: 8, // gap between adjacent chips
+              runSpacing: 0, // gap between lines
+              children: skillsAndInterest
+                  .map((i) => Chip(
+                        backgroundColor: AppColors.jungle,
+                        label: Text(
+                          i,
+                          style: Theme.of(context).textTheme.subtitle2,
+                        ),
+                      ))
+                  .toList(),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+/// PROFILE BOTTOM
+class _ProfileBottom extends StatelessWidget {
+  const _ProfileBottom({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(16.0),
+      child: Column(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              color: AppColors.jungle,
+              borderRadius: BorderRadius.all(Radius.circular(12)),
+            ),
+            child: Row(
+              children: [
+                Expanded(
+                  flex: 1,
+                  child: ClipRRect(
+                    borderRadius: BorderRadius.only(
+                      topLeft: Radius.circular(12.0),
+                      bottomLeft: Radius.circular(12.0),
+                    ),
+                    child: SvgPicture.asset(
+                      "assets/images/lotus.svg",
+                      alignment: Alignment.centerLeft,
+                      color: AppColors.canopy,
+                    ),
+                  ),
+                ),
+                Expanded(
+                  flex: 2,
+                  child: Padding(
+                    padding: const EdgeInsets.symmetric(vertical: 14.0),
+                    child: Column(
+                      children: [
+                        Row(
+                          children: [
+                            Text(
+                              'You are on the way from'.i18n,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .subtitle2HighEmphasis,
+                            ),
+                          ],
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.symmetric(vertical: 8.0),
+                          child:
+                              // If the text is longer than the condition use 2 rows instead of one
+                              ('Resident'.i18n.length > 8 &&
```

Comment:
I hate that we need to complicate the code with this because of the design. 
Im gonna talk to Jason and see if there is a better design we can use.